### ISSUE TITLE
Geocoder Locale is not working

### ISSUE THREAD

**Issue by soerenmartius** [egeloen/IvoryGoogleMapBundle#133]
Hi,

When i set the locale for geocoder in the config:
```

ivory_google_map:
    geocoder:
        enabled: true
        class: "Ivory\GoogleMap\Services\Geocoding\Geocoder"
        provider:
            class: "Ivory\GoogleMap\Services\Geocoding\GeocoderProvider"
            locale: "de"
            api_key: "foo"
    map:
        async: true
```
It won't work. The Geocoder always returns english formatted results.

**Comment by SergioEnrique** [comment]
Yes, I need these options. Also y would like to configure an specific "region" that limits results for my country

**Comment by egeloen** [comment]
Your issue has been fixed in #159 but will only be part of the 3.x version of the bundle.
